### ISSUE TITLE
"Cannot read property 'local' of undefined" when using --build-optimizer

### ISSUE THREAD

**Issue by maxime1992** [angular/angular-cli#7766]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
```bash
git clone https://github.com/maxime1992/compose-teams.git
cd compose-teams
git checkout f14f95983bab25
yarn
ng serve --prod --build-optimizer
```

### The log given by the failure.
```
Uncaught TypeError: Cannot read property 'local' of undefined
    at Object.cDNt (main.17b668cf96d0291fd30e.bundle.js:1)
    at n (inline.d6a1c3788979ee0bfff3.bundle.js:1)
    at Object.0 (main.17b668cf96d0291fd30e.bundle.js:1)
    at n (inline.d6a1c3788979ee0bfff3.bundle.js:1)
    at window.webpackJsonp (inline.d6a1c3788979ee0bfff3.bundle.js:1)
    at main.17b668cf96d0291fd30e.bundle.js:1
cDNt @ main.17b668cf96d0291fd30e.bundle.js:1
n @ inline.d6a1c3788979ee0bfff3.bundle.js:1
0 @ main.17b668cf96d0291fd30e.bundle.js:1
n @ inline.d6a1c3788979ee0bfff3.bundle.js:1
window.webpackJsonp @ inline.d6a1c3788979ee0bfff3.bundle.js:1
(anonymous) @ main.17b668cf96d0291fd30e.bundle.js:1
```

### Mention any other details that might be useful.
I'm not sure but it might be related to https://github.com/angular/angular-cli/issues/6196

The app is deployed on Github pages and it's broken right now in case you want to take a look into the issue: https://maxime1992.github.io/compose-teams

@filipesilva I talked with you few months ago when build-optimizer came out and I had a similar error that you fixed pretty quickly :clap: maybe you might see trough this one too :wink:

**Comment by clydin** [comment]
Can you try with the latest and recently released version of the build optimizer (0.0.21)?

**Comment by maxime1992** [comment]
@clydin it's working great! Thanks :) :tada: :clap:
